### PR TITLE
Add Juniper Networks ACX5048

### PR DESCRIPTION
**Pull request by ljarasius**
Hardware Guide - https://www.juniper.net/documentation/us/en/hardware/acx5000/acx5000.pdf

### CODE REVIEW

**Comment by ljarasius** [comment]
AC and DC models are hard-coded into the chassis despite having hot-swappable PSUs, therefore adding 2 separate device definitions.

File: device-types/Juniper/ACX5048-AC.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+manufacturer: Juniper
+model: ACX5048-AC
+slug: juniper-acx5048-ac
+part_number: ACX5048-AC
+u_height: 1
```
Comment (danner26) [inline device-types/Juniper/ACX5048-AC.yaml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Juniper/ACX5048-DC.yaml [hunk 2]
```diff
@@ -0,0 +1,141 @@
+---
+manufacturer: Juniper
+model: ACX5048-DC
+slug: juniper-acx5048-dc
+part_number: ACX5048-DC
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Juniper/ACX5048-DC.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Juniper/ACX5048-AC.yaml (hunk 1 above)
Comment (ljarasius) [reply device-types/Juniper/ACX5048-AC.yaml:7]: Hey @danner26,

I have added the `airflow` attribute as requested.

As for the device weight, the datasheet only provides a "loaded" weight with all module bays populated. Please let me know if you would like to include this, however I am of the understanding that a true device weight and weight for each individual module would be more accurate and therefore preferred.

Cheers,
Louis

File: device-types/Juniper/ACX5048-DC.yaml (hunk 2 above)
Comment (ljarasius) [reply device-types/Juniper/ACX5048-DC.yaml:8]: Hey @danner26,

I have added the `airflow` attribute as requested.

As for the device weight, the datasheet only provides a "loaded" weight with all module bays populated. Please let me know if you would like to include this, however I am of the understanding that a true device weight and weight for each individual module would be more accurate and therefore preferred.

Cheers,
Louis

**Comment by ljarasius** [comment]
Hi @danner26,

Any update here? Comments on your requested items are in the review.

Cheers,
Louis

File: device-types/Juniper/ACX5048-DC.yaml (hunk 2 above)
Comment (danner26) [reply device-types/Juniper/ACX5048-DC.yaml:8]: Hello, thanks for getting that! No we would want the unloaded weight so you can omit this

File: device-types/Juniper/ACX5048-AC.yaml (hunk 1 above)
Comment (danner26) [reply device-types/Juniper/ACX5048-AC.yaml:7]: Hello, thanks for getting that! No we would want the unloaded weight so you can omit this

**Review by danner26 — APPROVED** [review]

**Pull request merged**